webrtc/lib/webrtc/media_event.ex

```diff
@@ -7,16 +7,20 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.MediaEvent do
 
   @type t() :: map()
 
-  @spec connected(Endpoint.id(), list()) :: t()
-  def connected(endpoint_id, other_endpoints) do
+  @spec connected(Endpoint.id(), list(), %{
+          Endpoint.id() => %{Track.id() => %{enabled: boolean(), active_encodings: [any()]}}
+        }) :: t()
+  def connected(endpoint_id, other_endpoints, endpoint_id_to_simulcast_configs) do
     other_endpoints =
       other_endpoints
       |> Enum.map(
         &%{
           id: &1.id,
           type: to_type_string(&1.type),
           metadata: &1.metadata,
-          trackIdToMetadata: Endpoint.get_active_track_metadata(&1)
+          trackIdToMetadata: Endpoint.get_active_track_metadata(&1),
+          trackIdToSimulcastConfig:
+            endpoint_id_to_simulcast_configs |> Map.get(&1.id) |> map_simulcast_config()
```

Comment:
* I would opt for merging those fields as the more maps we have, the harder it is to read the code. However, we would probably have to take care of backward compatibility 

* Can't we use `other_endpoints` to get info about simulcast config? Every endpoint struct has a list of inbound tracks. If we need to, we can add required info to track's context but in some other place we assume that tracks with single `:high` variant are non-simulcast while the others are simulcast https://github.com/jellyfish-dev/membrane_rtc_engine/blob/49ca083eef8d168e74f675c3b9167097d59a9269/webrtc/lib/webrtc_endpoint.ex#L738-L740

Reply:
For the first point, I thought about it, but I didn't find a way to merge these maps and also be backward compatible. If we merge it and create map `trackIdToTrackInfo`, it will break compatibility. If you thought about sending both `trackIdToMetadata` and `trackIdToTrackInfo,` I think this doesn't make it easier to read.

For the second point here, we could use `other_endpoints,` but we can't use it in `handle_child_notification` `:new_tracks`, so I decided to create it in the same way in both functions, but the only way to achieve that was by checking a number of variants.